app/components/UI/SnapsExecutionWebView/SnapsExecutionWebView.tsx

```diff
@@ -35,13 +35,18 @@ const SnapsExecutionWebView = () => {
     stream?._onMessage(data);
   };
 
+  // prod: https://gantunesr.github.io/mobile-execution-environment/
+  // iOS: http://localhost:3001/mobile-execution-environment
+  // android: http://10.0.2.2:3001/mobile-execution-environment
+  const envURI = 'https://gantunesr.github.io/mobile-execution-environment/';
+
   return (
     <ScrollView style={styles.container}>
       <View style={styles.webview}>
         <WebView
           ref={webviewRef}
           source={{
-            uri: 'https://gantunesr.github.io/mobile-execution-environment/',
+            uri: envURI,
```

Comment:
In order with the previous comment,

```suggestion
            uri: envURI.prod,
```

Reply:
great idea